cms/static/js/views/modals/move_xblock_modal.js

```diff
@@ -105,6 +125,53 @@ function($, Backbone, _, gettext, BaseView, BaseModal, XBlockInfoModel, MoveXBlo
                     ancestorInfo: ancestorInfo
                 }
             );
+        },
+
+        updateMoveState: function(isValidMove) {
+            var $moveButton = this.$el.find('.action-move');
+            if (isValidMove) {
+                $moveButton.removeAttr('disabled').removeClass('is-disabled');
```

Comment:
since this is not actually a button, it cannot have a disabled attribute.  It is currently not doing anything.  This will need to be a proper button for this to work.